tests/languages/birb/string_feature.test

```diff
@@ -0,0 +1,13 @@
+"" ''
+"fo\"o" 'fo\'o'

```

Comment:
Please add a multiline string example.

This also leads to another point: C-like strings aren't multiline. You need to make your own pattern like this: `/("|')(?:\\[\s\S]|(?!\1)[^\\])*\1/`